daemon/src/routes_taker.rs

```diff
@@ -299,4 +300,13 @@ mod tests {
 
         assert!(serde_json::from_str::<WalletRestoreRequest>(json).is_ok());
     }
+
+    #[test]
+    fn mnemonic_serializes_to_space_seperated_word_list() {
+        let mnemonic =
+            Mnemonic::from_str("clay logic upset stadium treat nothing lumber seek inject require tourist disagree atom call lady eagle caught equip dove game connect peanut noble quick").unwrap();
+        let json = serde_json::to_string(&mnemonic).unwrap();
+
+        assert_eq!(json, "\"clay logic upset stadium treat nothing lumber seek inject require tourist disagree atom call lady eagle caught equip dove game connect peanut noble quick\"");
```

Comment:
So we are sending a quoted string in the HTTP body back to the client? That is kinda odd and sort of defeats the point of using JSON. Why not send an object instead?

Reply:
There are two endpoints which would return this JSON object. Would you reuse the same type or make two different types.

Reply:
Yeah same is fine. Can even use the same one from the request. As long as there is a test that freezes the impl, that is all we need.